dietpi/func/dietpi-globals

```diff
@@ -1416,10 +1416,8 @@ $print_logfile_info
 		if (( $G_HW_MODEL < 10 )); then
 
 			G_AG_CHECK_INSTALL_PREREQ rpi-update
-			export SKIP_BACKUP=1
-			export SKIP_WARNING=1
-			G_RUN_CMD rpi-update
-			apt-mark hold raspberrypi-bootloader raspberrypi-kernel libraspberrypi-bin
+			SKIP_BACKUP=1 SKIP_WARNING=1 G_RUN_CMD rpi-update #PRUNE_MODULES=1 would remove old/unused /lib/modules/* directories, too dangerous if update fails/firmware is unstable?
```

Comment:
Nice 👍 

I did try this, but after ```G_RUN_CMD SKIP_BACKUP=1 SKIP_WARNING=1```, thats why it failed 😄

Reply:
@Fourdee 
Jep, also tried it the other way round first, then also was not sure if variables are forwarded through G_RUN_CMD to sub command, but it works 😃.